### PR TITLE
ovs_qos: decode process.system_output()

### PR DESCRIPTION
**Pull request by sitoliu**
ID: 1649159
Signed-off-by: Sitong Liu <siliu@redhat.com>

### CODE REVIEW

File: qemu/tests/ovs_qos.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def set_ovs_port_attr(iface, attribute, value):
         """
         cmd = "ovs-vsctl set interface %s %s=%s" % (iface, attribute, value)
         logging.info("execute host command: %s" % cmd)
-        status = process.system(cmd, ignore_status=True)
+        status = process.system(cmd, ignore_status=True).decode()
```
Comment (pezhang) [inline qemu/tests/ovs_qos.py]: @sitoliu process.system() returns an "int" value. Hit below error:

23:22:39 ERROR|   File "/var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-qemu/qemu/tests/ovs_qos.py", line 40, in set_ovs_port_attr
    status = process.system(cmd, ignore_status=True).decode()
23:22:39 ERROR| AttributeError: 'int' object has no attribute 'decode'

**Comment by pezhang** [comment]
Test pass.

Looks good to me.
Acked by: Pei Zhang pezhang@redhat.com

File: qemu/tests/ovs_qos.py [hunk 2]
```diff
@@ -131,6 +131,8 @@ def __get_vminfo():
             """
             login_timeout = float(params.get("login_timeout", 360))
             clear_iptables_cmd = "service iptables stop; iptables -F"
+            stop_firewall_cmd = "systemctl stop firewalld||"
+            stop_firewall_cmd += "service firewalld stop"
```
Comment (xiywang) [inline qemu/tests/ovs_qos.py]: "iptalbes -F" are also used to stop firewall.
You can use:
clear_iptables_cmd = "systemctl stop firewalld || service iptables stop || iptables -F"

File: qemu/tests/ovs_qos.py [hunk 3]
```diff
@@ -142,6 +144,7 @@ def __get_vminfo():
                 vm.verify_alive()
                 session = vm.wait_for_login(timeout=login_timeout)
                 session.cmd(clear_iptables_cmd, ignore_all_errors=True)
+                session.cmd(stop_firewall_cmd, ignore_all_errors=True)
```
Comment (xiywang) [inline qemu/tests/ovs_qos.py]: Drop this line if you add firewalld stop into clear_iptables_cmd

**Pull request merged**